log/sequencer.go

```diff
@@ -110,20 +111,20 @@ func (s Sequencer) sequenceLeaves(mt *merkle.CompactMerkleTree, leaves []trillia
 	return nodeMap, leaves, nil
 }
 
-func (s Sequencer) initMerkleTreeFromStorage(currentRoot trillian.SignedLogRoot, tx storage.LogTX) (*merkle.CompactMerkleTree, error) {
+func (s Sequencer) initMerkleTreeFromStorage(ctx context.Context, currentRoot trillian.SignedLogRoot, tx storage.LogTX) (*merkle.CompactMerkleTree, error) {
 	if currentRoot.TreeSize == 0 {
 		return merkle.NewCompactMerkleTree(s.hasher), nil
 	}
 
 	// Initialize the compact tree state to match the latest root in the database
-	return s.buildMerkleTreeFromStorageAtRoot(currentRoot, tx)
+	return s.buildMerkleTreeFromStorageAtRoot(ctx, currentRoot, tx)
 }
 
-func (s Sequencer) signRoot(root trillian.SignedLogRoot) (trillian.DigitallySigned, error) {
+func (s Sequencer) createRootSignature(ctx context.Context, root trillian.SignedLogRoot) (trillian.DigitallySigned, error) {
 	signer, err := s.keyManager.Signer()
 
 	if err != nil {
-		glog.Warningf("key manager failed to create crypto.Signer: %v", err)
+		glog.Warningf("%skey manager failed to create crypto.Signer: %v", util.LogIDPrefix(ctx), err)
```

Comment:
Having the %sjoined with the following message is causing me a horrible mental itch :)
Could we squander some bytes and have it be "%s: …"?